web-ui/src/components/reviews/periods/ReviewPeriodCard.jsx

```diff
@@ -0,0 +1,214 @@
+import PropTypes from 'prop-types';
+import React, { useCallback, useContext, useEffect, useState } from 'react';
+
+import {
+  BorderColor,
+  DoorFront,
+  HourglassTop,
+  MeetingRoom,
+  QuestionMark
+} from '@mui/icons-material';
+
+import {
+  Avatar,
+  Card,
+  CardActions,
+  CardContent,
+  Collapse,
+  ListItemAvatar,
+  ListItemText
+} from '@mui/material';
+
+import ExpandMore from '../../expand-more/ExpandMore';
+import { resolve } from '../../../api/api.js';
+import { AppContext } from '../../../context/AppContext';
+import {
+  selectCsrfToken,
+  selectCurrentMembers,
+  selectHasUpdateReviewAssignmentsPermission,
+  selectReviewPeriods
+} from '../../../context/selectors';
+import { titleCase } from '../../../helpers/strings.js';
+
+import './ReviewPeriodCard.css';
+
+const propTypes = {
+  mode: PropTypes.string,
+  onSelect: PropTypes.func.isRequired,
+  periodId: PropTypes.string.isRequired,
+  selfReviews: PropTypes.object.isRequired
+};
+const displayName = 'ReviewPeriodCard';
+
+const ReviewStatus = {
+  PLANNING: 'PLANNING',
+  AWAITING_APPROVAL: 'AWAITING_APPROVAL',
+  OPEN: 'OPEN',
+  CLOSED: 'CLOSED',
+  UNKNOWN: 'UNKNOWN'
+};
+
+const reviewStatusIconMap = {
+  [ReviewStatus.PLANNING]: <BorderColor />,
+  [ReviewStatus.AWAITING_APPROVAL]: <HourglassTop />,
+  [ReviewStatus.OPEN]: <MeetingRoom />,
+  [ReviewStatus.CLOSED]: <DoorFront />,
+  [ReviewStatus.UNKNOWN]: <QuestionMark />
+};
+
+const ReviewPeriodCard = ({ mode, onSelect, periodId, selfReviews }) => {
+  const { state } = useContext(AppContext);
+  const [approvalStats, setApprovalStats] = useState([]);
+  const [overallApprovalPercentage, setOverallApprovalPercentage] = useState(0);
+  const [expanded, setExpanded] = useState(false);
+
+  const csrf = selectCsrfToken(state);
+  const currentMembers = selectCurrentMembers(state);
+  const canUpdateReviewAssignments =
+    selectHasUpdateReviewAssignmentsPermission(state);
+  const periods = selectReviewPeriods(state);
+
+  const period = periods.find(period => period.id === periodId);
+  const showPercentages =
```

Comment:
This determines whether we should show percentages. The user must have a certain permission AND the review period must have a status of "AWAITING APPROVAL".